client/sites-dashboard/components/sites-plan-renew-nag.tsx

```diff
@@ -7,8 +7,9 @@ import { sprintf } from '@wordpress/i18n';
 import { useI18n } from '@wordpress/react-i18n';
 import { useEffect } from 'react';
 import { useInView } from 'react-intersection-observer';
-const PLAN_RENEW_NAG_IN_VIEW = 'calypso_sites_dashboard_plan_renew_nag_inview';
-const PLAN_RENEW_NAG_ON_CLICK = 'calypso_sites_dashboard_plan_renew_nag_click';
+
```

Comment:
While we're touching this context, can we remove `react-intersection-observer` ?

Reply:
Sure !, done.